### DISCUSSION TITLE
possibility to get remote notifications/mails or auto-upload a picture/csv to ftp/cloud/etc directly from android app

### DISCUSSION THREAD

**Discussion by h3rm** [emericg/WatchFlower#38] [Ideas]
Hi! 

At the moment i use VNC to connect to my android(server) device with VPN activated on my router and the VNC-Server-Android app "alpha vnc lite" (https://play.google.com/store/apps/details?id=de.abr.android.avnc&hl=de) where all the (xiaomi flower care) sensors are configured, named and have there "place" and min/max-settings of soil moisture/temp/EC/lux configured.

Would it be possible to integrate some kind of remote notfication outside my network like "motioneye os" (https://github.com/ccrisan/motioneyeos/wiki/Features) for raspberry pi provides? There you have to possibility to get mails (you have to use user/pw from one of your own mail accounts) or/and it is possible to upload pictures on google drive/dropbox (https://github.com/ccrisan/motioneyeos/wiki/Synchronize-with-Google-Drive) or upload pictures on your own ftp-server (https://imgur.com/ICWhQ4Q).

Would be really cool when watchflower for android (the windows 10 app is a little bit unstable sometimes and often find the sensors only when i manually search for bluetooth devices with the windows-integrated bluetooth-feature "find devices" a few seconds before -> otherwise the app crashes or can't find all flower care sensors - so i bought the "alpha vnc lite" android app, the watchflower android app and a cheap ulephone android 10 device and now it works very much better and very stable and i donÄt have to let my notebook active 24/7.... :) ) can upload pictures of the main interface (or if it's not possible all sensor data from each sensor - with the defined name of the plant and "the defined place" - in a csv/txt-file) somewhere automatically? Maybe also with the option to decide how often i want to upload files or pix/send mails or whatever is possible to program for you... ;)

thank you again! :)

**Comment by emericg** [comment]
Hi,
At the moment no there is no way to do any of that.
The Android version doesn't even have a background daemon and notifications.

Now I guess there would be a couple of things possible in the future, like more notifications option for desktops (including email, and sending the CSV for instance).
There is another possibility, it's to let a raspberry with WatchFlower working. There is a "no UI" mode for the desktop.
Also it should be possible to send the actual data on a MySql database (internal or online), and to do a small web UI with this db (if you really want to check your data outside of your network, that could be a solution). That's not completely working at the moment, but it has been tested and it will be a thing someday ^^
